po/zh_CN.po

```diff
@@ -49,12 +49,71 @@ msgstr ""
 msgid "Whether watchdog integration is enabled"
 msgstr "是否启用看门狗集成设置"
 
-#: tools/crm_resource.c:1418
+#: tools/crm_resource.c:1388
 #, fuzzy, c-format
 msgid "Metadata query for %s failed: %s"
 msgstr "，查询%s的元数据失败: %s\n"
 
-#: tools/crm_resource.c:1424
+#: tools/crm_resource.c:1394
 #, c-format
 msgid "'%s' is not a valid agent specification"
 msgstr "'%s' 是一个无效的代理"
+
+#: daemons/fenced/pacemaker-fenced.c:1478
+#, c-format
+msgid ""
+" <longdesc lang=\"en\">Instance attributes available for all \"stonith\"-"
+"class resources and used by Pacemaker's fence daemon, formerly known as "
+"stonithd</longdesc>\n"
+msgstr ""
+" <longdesc lang=\"zh\">实例属性可用于所有stonith类资源，并由Pacemaker的fence"
+"守护程序使用（以前称为stonithd）</longdesc>\n"
+
+#: daemons/fenced/pacemaker-fenced.c:1480
+#, c-format
+msgid ""
+" <shortdesc lang=\"en\">Instance attributes available for all \"stonith\"-"
+"class resources</shortdesc>\n"
+msgstr ""
+" <shortdesc lang=\"zh\">可用于所有stonith类资源的实例属性</shortdesc>\n"
+
+#: daemons/fenced/pacemaker-fenced.c:1495
+#, c-format
+msgid ""
+"    <longdesc lang=\"en\">Some devices do not support the standard 'port' "
+"parameter or may provide additional ones.\n"
+"Use this to specify an alternate, device-specific, parameter that should "
+"indicate the machine to be fenced.\n"
+"A value of 'none' can be used to tell the cluster not to supply any "
+"additional parameters.\n"
+"     </longdesc>\n"
+msgstr ""
+"    <longdesc lang=\"zh\">某些设备可能不支持使用标准的端口参数，也可能会提供额外的端口参数。\n"
+"使用此参数可以为需要防护（fence）的机器指定一个备用的，专用于该设备的参数,该参数应指出要fence的机器。\n"
+"使用值none可用来告诉集群不提供任何额外的参数\n"
+"     </longdesc>\n"
+
+#: daemons/fenced/pacemaker-fenced.c:1500
+#, c-format
+msgid ""
+"    <shortdesc lang=\"en\">Advanced use only: An alternate parameter to "
+"supply instead of 'port'</shortdesc>\n"
+msgstr ""
+"    <shortdesc lang=\"zh\">Advanced use only:(仅限高级使用)备用参数可替代'port'</shortdesc>\n"
```

Comment:
Is it intentional to repeat "Advanced use only" here given that it's already translated?